ui/helpers/utils/notification.util.ts

```diff
@@ -260,8 +260,11 @@ export const getUsdAmount = (amount: string, decimals: string, usd: string) => {
     return '';
   }
 
-  const amountInEther = getAmount(amount, decimals);
-  const numericAmount = parseFloat(amountInEther) * parseFloat(usd);
+  const amountInEther = calcTokenAmount(
+    amount,
+    parseFloat(decimals),
+  ).toNumber();
```

Comment:
My only concern is how this is a different method compared to portfolio. 

In portfolio we are using Ethers+BigNumber.js so little conversion loss.
But extension may have some formatting.

... we can deal with this in the future lolol